From 81cef0394465cf8921297d2e55bde6dd80911cef Mon Sep 17 00:00:00 2001
From: Bernhard <rep.dot.nop@gmail.com>
Date: Sat, 25 Sep 2021 16:49:27 -0500
Subject: [PATCH] ntpdate: Fix -d option

ntpdate is documented as accepting option "-d" but does in fact reject
it.

Signed-off-by: Richard Laager <rlaager@wiktel.com>
---
 attic/ntpdate | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/attic/ntpdate b/attic/ntpdate
index 0c867d544..acb22c1bf 100755
--- a/attic/ntpdate
+++ b/attic/ntpdate
@@ -62,7 +62,7 @@ TIMEOUT="-t 1"
 setclock=yes
 echo=no
 log=no
-while getopts 46a:bBe:hk:no:p:qst:uv opt
+while getopts 46a:bBde:hk:no:p:qst:uv opt
 do
     case $opt in
 	4) PASSTHROUGH="$PASSTHROUGH -4";;
-- 
2.33.0

